stable/dask/README.md

```diff
@@ -2,38 +2,41 @@
 
 Dask allows distributed computation in Python.
 
--  https://dask.pydata.org
--  http://jupyter.org/
-
+- <https://dask.org>
+- <https://jupyter.org/>
 
 ## Chart Details
 
 This chart will deploy the following:
 
--   1 x Dask scheduler with port 8786 (scheduler) and 80 (Web UI) exposed on an external LoadBalancer
--   3 x Dask workers that connect to the scheduler
--   1 x Jupyter notebook (optional) with port 80 exposed on an external LoadBalancer
--   All using Kubernetes Deployments
+- 1 x Dask scheduler with port 8786 (scheduler) and 80 (Web UI) exposed on an external LoadBalancer (default)
+- 3 x Dask workers that connect to the scheduler
+- 1 x Jupyter notebook (optional) with port 80 exposed on an external LoadBalancer (default)
+- All using Kubernetes Deployments
+
+> **Tip**: See the [Kubernetes Service Type Docs](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)
+for the differences between ClusterIP, NodePort, and LoadBalancer.
 
 ## Installing the Chart
 
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm install --name my-release stable/dask
+helm install --name my-release stable/dask
 ```
 
```

Comment:
Add after the command: "Depending on how your cluster was setup, you may also need to specify --namespace my-namespace"